src/SIL.XForge.Scripture/ClientApp/src/app/translate/editor/lynx/insights/quill-services/quill-editor-ready.service.spec.ts

```
@@ -0,0 +1,89 @@
+class MockQuillEditor {
+  private eventHandlers: Record<string, Function[]> = {};
+  private contentLength = 0;
+
+  emitEditorChange(eventName: string): void {
+    if (this.eventHandlers['editor-change']) {
+      this.eventHandlers['editor-change'].forEach(handler => handler(eventName, {}));
+    }
+  }
+
+  // Compatible with rxjs 'fromEvent' (EventTarget-like)
+  addEventListener(eventName: string, callback: Function): void {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)